README.md

```diff
@@ -61,9 +76,11 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
```

Comment:
- [ ] Hi @indigodavid, great job :+1: for having a descriptive README.md file. Please let's equally give appropriate credit to the author of the original design(Gregoire Vella on Behance) as required by the [Creative Commons license of the design.](https://creativecommons.org/licenses/by-nc/4.0/)

Reply:
Thank you for noticing it. I did the change as requested.